comment/api/serializers.py

```diff
@@ -22,10 +25,14 @@ def get_profile_model():
 def get_user_fields():
     user_model = get_user_model()
     fields = user_model._meta.get_fields()
+    api_fields = list(settings.COMMENT_USER_API_FIELDS)
+    api_fields.append('profile')
```

Comment:
Why are you appending the `profile` field here when you have already added it to the [defaults](https://github.com/Radi85/Comment/pull/126/files#diff-04e6bdb3bd4b29a0735aa015d5172ee65fa44d84db8c904db1aa157adca2fe46R7)?

As far I can understand, you might not want to change these fields here?